src/lib/taxaFetchAPI.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import inatjs from "inaturalistjs";
+
+const ANCESTOR_FIELDS = {
+  name: true,
+  preferred_common_name: true,
+  rank: true
+};
+
+const PHOTO_FIELDS = {
+  id: true,
+  attribution: true,
+  license_code: true,
+  url: true
+};
+
+const FIELDS = {
+  ancestors: ANCESTOR_FIELDS,
+  name: true,
+  preferred_common_name: true,
+  rank: true,
+  taxon_photos: {
+    photo: PHOTO_FIELDS
+  },
+  wikipedia_summary: true,
+  wikipedia_url: true
+};
+
+const fetchTaxa = async ( id: number ): Promise<any> => {
+  const params = {
+    fields: FIELDS
+  };
```

Comment:
`params` never changes so you could probably statically define it outside of the function like the rest of the fields stuff.